docs/release-notes/v4.md

```diff
@@ -1,15 +1,15 @@
-# Essential Next.js plugin v4 release notes
+# Next Runtime v4 release notes
```

Comment:
[pebble] I think we should leave this page and its content as Essential Next.js plugin. When v4 happened, that's what it was named. But the updated URLs are necessary to not break links. 😄